janus_server/src/datastore.rs

```diff
@@ -823,27 +823,26 @@ impl Transaction<'_> {
         )
     }
 
-    /// If a collect job corresponding to the provided values exists, its UUID
-    /// is returned, which may then be used to construct a collect job URI. If
-    /// that collect job does not exist, then a UUID is generated and the
-    /// collect job is stored, and the UUID is returned.
+    /// If a collect job corresponding to the provided values exists, its UUID is returned, which
+    /// may then be used to construct a collect job URI. If that collect job does not exist, returns
+    /// `Ok(None)`.
     #[tracing::instrument(skip(self, encoded_aggregation_parameter), err)]
-    pub(crate) async fn lookup_collect_job_uri(
+    pub(crate) async fn get_collect_job_uuid(
         &self,
         task_id: TaskId,
         batch_interval: Interval,
         encoded_aggregation_parameter: &[u8],
-    ) -> Result<Uuid, Error> {
+    ) -> Result<Option<Uuid>, Error> {
         let batch_interval_start = batch_interval.start.as_naive_date_time();
         let batch_interval_duration = i64::try_from(batch_interval.duration.0)?;
 
         let stmt = self
             .tx
             .prepare_cached(
                 "SELECT collect_job_id FROM collect_jobs
-            WHERE task_id = (SELECT id FROM tasks WHERE task_id = $1)
-            AND batch_interval_start = $2 AND batch_interval_duration = $3
-            AND aggregation_param = $4",
+                WHERE task_id = (SELECT id FROM tasks WHERE task_id = $1)
+                    AND batch_interval_start = $2 AND batch_interval_duration = $3
+                    AND aggregation_param = $4",
```

Comment:
```suggestion
                AND batch_interval_start = $2 AND batch_interval_duration = $3
                AND aggregation_param = $4",
```

Reply:
Sounds like we need an inline SQL linter! Done.